ui/src/app/views/workspace-library.tsx

```diff
@@ -95,22 +99,41 @@ export const WorkspaceLibrary = withUserProfile()
   updateListedWorkspaces(tabEnum) {
     switch (tabEnum) {
       case tabEnum === libraryTabEnums.PUBLISHED_WORKSPACES:
-        this.reloadPublishedWorkspaces();
+        this.loadPublishedWorkspaces();
         break;
       default:
+        this.loadFeaturedWorkspaces();
         break;
     }
   }
 
-  async reloadPublishedWorkspaces() {
-    this.setState({workspacesLoading: true});
+  async loadFeaturedWorkspaces() {
+    try {
+      const resp = await featuredWorkspacesConfigApi().getFeaturedWorkspacesConfig();
+      const idToNamespace = new Map();
+      resp.featuredWorkspacesList.map(fw => idToNamespace.set(fw.id, fw.namespace));
+
+      this.setState({
+        featuredWorkspaces: this.state.workspaceList.filter(ws => idToNamespace.get(ws.workspace.id)
```

Comment:
Not an issue for this PR, but I would have hoped / thought we could start ignoring workspace namespaces for logic like this, at least in a post-1ppw world (since the namespace should be equal to the project ID).

But maybe this isn't the case? @ericsong maybe you have clarity on this one... do we need to ~forever support projects which were created pre-1PPW, or will we eventually be able to drop the "namespace" concept?

Reply:
it was unclear to me. my impression is that workspace ids were only unique across namespaces

Reply:
We discussed doing this but there are complications and it was not in scope for 1ppw. As of today, you need both namespace and name to create a link. We don't need to forever support old project/workspaces - after https://precisionmedicineinitiative.atlassian.net/browse/RW-2705 we will more freedom to change this behavior if desired, but it needs more thought.

My current position is to keep the current behavior, though we could discuss further:
- Just using name as-is in the URL doesn't work, it's not unique across all projects
- Just using namespace is possible, but it removes the user-generated "id" from the URL, which I'd consider a net loss, or neutral change. When factoring in the extra lookups / client code needed to support that, I'd argue against it.
- (most likely path) Forcing uniqueness on names OR creating a new third globally unique namespace I don't consider to be worth the effort as it puts us in a position of policing a global namespace; it's possible but it's an annoying constraint to enforce which would only be applicable within AoU. It would involve a lot of code changes as we wouldn't be able to directly call into any Terra APIs without first going through our own lookup layer to determine the project.